lib/random-schedule.js

```diff
@@ -0,0 +1,74 @@
+const Random = require('random-js')
+const random = new Random(Random.engines.browserCrypto)
+const Scheduler = require('azure-arm-scheduler')
+const randString = require('crypto-random-string')
+
+exports.calcSeconds = function (timestamp) {
+  var stamp = timestamp.split(':')
+  return 3600 * stamp[0] + 60 * stamp[1]
+}
+
+exports.calcRandomSecond = function (startTime, endTime) {
+  return random.integer(startTime, endTime)
+}
+
+exports.dayShortToLong = function (inputDays) {
+  var days = []
+  for (var i = 0; i < inputDays.length; i++) {
+    var whatDay = inputDays[i].toUpperCase()
+    switch (whatDay) {
```

Comment:
I feel like this switch block could be replaced with a `map`-like data structure

```
const dayMap = {
  M: 'Monday',
  T: 'Tuesday',
  ...
}

days[i] = dayMap[whatDay]
```